blazor/dropdown-list/style.md

```diff
@@ -56,54 +112,20 @@ Use the following CSS to customize the focusing color of outline theme.
 }
 ```
 
-## Customizing the disabled component’s text color
-
-Use the following CSS to customize the text color when the component is disabled.
-
-```css
-.e-input-group.e-control-wrapper .e-input[disabled] {
-    -webkit-text-fill-color: #0d9133;
-}
-```
+You can use `e-outline` to the [CssClass](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_CssClass) property to achieve outline theme
 
-## Customizing the float label element's focusing color
+{% highlight cshtml %}
 
-Use the following CSS to customize the focusing color of float label element.
-
-```css
-.e-float-input.e-input-group:not(.e-float-icon-left) .e-float-line::before,.e-float-input.e-control-wrapper.e-input-group:not(.e-float-icon-left) .e-float-line::before,.e-float-input.e-input-group:not(.e-float-icon-left) .e-float-line::after,.e-float-input.e-control-wrapper.e-input-group:not(.e-float-icon-left) .e-float-line::after {
-    background-color: #2319b8;
-}
-
-.e-ddl.e-lib.e-input-group.e-control-wrapper.e-control-container.e-float-input.e-input-focus .e-float-text.e-label-top {
-    color: #2319b8;
-}
-```
-
-## Customizing the color of the placeholder text
-
-Use the following CSS to customize the text color of placeholder.
-
-```css
-.e-ddl.e-input-group input.e-input::placeholder {
-    color: red;
-}
-```
+{% include_relative code-snippet/style/outline-theme.razor %}
 
-## Customizing the placeholder to add mandatory indicator(*)
+{% endhighlight %}
 
-Use the following CSS to add the mandatory indicator * to the float label element.
+![Blazor DropDownList with different font family](./images/style/blazor_dropdown_outline-theme.png)
 
-```css
-.e-input-group.e-control-wrapper.e-control-container.e-float-input .e-float-text::after {
-    content: "*";
-    color: red;
-}
-```
 
 ## Customizing the background color of focus, hover, and active items
 
-Use the following CSS to customize the background color of focus, hover, and active items.
+Use the following CSS to customize the background color of `focus`, `hover`, and `active` items.
```

Comment:
Use the following CSS to customize the background color of the `focus,` `hover,` and `active` items.